.clj-kondo/config.edn

```diff
@@ -51,32 +51,34 @@
   ;; :shadowed-var                          {:level :warning}
   :deprecated-var
   {:exclude
-   {metabase.cmd/dump                                                            {:namespaces ["metabase\\.cmd-test" "metabase-enterprise\\.serialization\\.cmd-test"]}
-    metabase.cmd/load                                                            {:namespaces ["metabase\\.cmd-test" "metabase-enterprise\\.serialization\\.cmd-test"]}
+   {metabase.cmd/dump                                                            {:namespaces ["metabase.cmd-test" "metabase-enterprise.serialization.cmd-test"]}
+    metabase.cmd/load                                                            {:namespaces ["metabase.cmd-test" "metabase-enterprise.serialization.cmd-test"]}
+    metabase.api.common/defendpoint-schema                                       {:namespaces ["metabase.api.*" "metabase-enterprise.*.api" "metabase.util.schema-test"]}
+    metabase.api.common/defendpoint-async-schema                                 {:namespaces ["metabase.api.*"]}
     metabase.db.data-migrations/data-migrations                                  {:namespaces ["metabase.db.data-migrations"]}
     metabase.db.data-migrations/defmigration                                     {:namespaces ["metabase.db.data-migrations"]}
     metabase.db.data-migrations/run-migration-if-needed!                         {:namespaces ["metabase.db.data-migrations"]}
-    metabase.driver.common/ThreadSafeSimpleDateFormat                            {:namespaces ["metabase\\.driver.*"]}
-    metabase.driver.common/create-db-time-formatters                             {:namespaces ["metabase\\.driver.*"]}
+    metabase.driver.common/ThreadSafeSimpleDateFormat                            {:namespaces ["metabase.driver.*"]}
+    metabase.driver.common/create-db-time-formatters                             {:namespaces ["metabase.driver.*"]}
```

Comment:
I guess this ultimately make any difference, but a string here it gets interpreted as a regex -- https://github.com/clj-kondo/clj-kondo/blob/master/doc/linters.md#deprecated-var -- so `metabase.driver` would match something like `metabase-driver` or `metabaseydriver` in addition to `metabase.driver`. That's probably a nonsense scenario since we don't have any namespaces like that but just wanted to point out and argument in favor of the old version

Reply:
ah right. I think it's better to keep the original version. I'll switch it back.